kubernetes/samples/scripts/create-rcu-schema/drop-rcu-schema.sh

```diff
@@ -65,11 +63,13 @@ if [ -z ${namespace} ]; then
 fi
 
 if [ -z ${sysPassword} ]; then
```

Comment:
This line will fail if the `sysPassword` variable is empty.

Reply:
Six other lines have the same problem in this script; I have pushed a change to correct them.

And there are literally hundreds more across more than 30 other files. Search our "*.sh" files for "-z $"